src/apps/users/services/core.py

```diff
@@ -66,8 +74,18 @@ async def send_password_recovery(self, schema: PasswordRecoveryRequest) -> Publi
 
         exp = settings.authentication.password_recover.expiration // 60
 
+        # Default to web frontend
+        frontend_base_url = settings.service.urls.frontend.web_base
```

Comment:
rename this var to frontend_base_domain, it always have only the domain

Reply:
Same comment as [here](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/pull/1100#discussion_r1491718601)